tools/lsp/ui/components/selection-popup.slint

```diff
@@ -141,12 +141,17 @@ component PopupInner inherits Rectangle {
     border-color: lightgray;
 
     width: 250px;
+    init => {
+        filter-edit.focus();
+    }
```

Comment:
I am fine to move this here, but please delete the code in lint 169++ when you do.